src/Microsoft.AspNet.Mvc.Razor.Host/Properties/AssemblyInfo.cs

```diff
@@ -1,6 +1,8 @@
-﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
Actually we're removing the BOM. Found that our assemblyinfo.cs files are extreeemly inconsistent (almost seems like a 50/50 split) with their bom's, decided to remove all of them.